web.go

```diff
@@ -38,72 +39,63 @@ func (fh *ForwardHandler) Handle(w http.ResponseWriter, r *http.Request) {
 	switch eventType {
 	case "api_post_event", "deployment_info":
 		log.WithField("eventType", eventType).Info("handling event")
-		fh.HandleAppEvent(w, body)
+		err = fh.HandleAppEvent(body)
 	case "app_terminated_event":
 		log.WithField("eventType", "app_terminated_event").Info("handling event")
-		fh.HandleTerminationEvent(w, body)
+		err = fh.HandleTerminationEvent(body)
 	case "status_update_event":
 		log.WithField("eventType", "status_update_event").Info("handling event")
-		fh.HandleStatusEvent(w, body)
+		err = fh.HandleStatusEvent(body)
 	default:
 		log.WithField("eventType", eventType).Info("not handling event")
 		w.WriteHeader(200)
 		fmt.Fprintf(w, "cannot handle %s\n", eventType)
+		return
 	}
-	log.Debug(string(body))
-}
 
-func (fh *ForwardHandler) HandleAppEvent(w http.ResponseWriter, body []byte) {
-	event, err := events.ParseEvent(body)
 	if err != nil {
 		w.WriteHeader(500)
 		fmt.Fprintln(w, err.Error())
 		log.Printf("[ERROR] body generated error: %s", err.Error())
```

Comment:
while you'er cleaning up the error handling here, could you change this to a `log.Error` instead? I'm not sure why this is this way.